redash/models/__init__.py

```diff
@@ -130,6 +133,17 @@ def to_dict(self):
         }
 
 
+@python_2_unicode_compatible
+class TableMetadataQueriesLink(db.Model):
+    table_id = Column(db.Integer, db.ForeignKey("table_metadata.id", ondelete="CASCADE"), primary_key=True)
+    query_id = Column(db.Integer, db.ForeignKey("queries.id", ondelete="CASCADE"), primary_key=True)
+
+    __tablename__ = 'tablemetadata_queries_link'
```

Comment:
The naming scheme doesn't really match the name of the TableMetadata table name, `table_metadata`... and I'm not a big fan of the naming scheme of concatenated names for association tables.

The idiomatic meaning of the association table is more like `DataSourceExample`?

Reply:
hmmm... but the association table itself doesn't contain the data source example. It's basically just a mapping between a table_metadata id and a query id...

Maybe we can change it to `table_metadata_query_mapping`?

Reply:
Hmm, yeah, fair enough, let's do that and rename the class as well, please.